diff --git a/android/build.gradle b/android/build.gradle
index 767eb43d1d9329215c12291c55034c964bcb059c..ba8d6b1298351a06beff6c40160d47549abb5be3 100644
--- a/android/build.gradle
+++ b/android/build.gradle
@@ -111,7 +111,6 @@ dependencies {
   //noinspection GradleDynamicVersion
   implementation "com.facebook.react:react-native:+"
   implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
-  implementation "com.facebook.fbjni:fbjni:+"
 }
 
 if (isNewArchitectureEnabled()) {
